src/Summoner/CLI.hs

```diff
@@ -41,6 +46,31 @@ summon = execParser prsr >>= runCommand
 runCommand :: Command -> IO ()
 runCommand = \case
     New opts -> runNew opts
+    ShowInfo opts -> runShow opts
+
+runShow :: ShowOpts -> IO ()
+runShow opts =
+    case opts of
+        -- show list of all available GHC versions
+        GhcList -> ghcVers
+        -- show a list of all available licenses
+        LicenseList -> licenses
+            -- show a specific license
+        LicenseText name ->
+            case parseLicenseName (toText name) of
+                Nothing -> do
+                    errorMessage "This wasn't a valid choice."
+                    infoMessage "Here is the list of supported licenses"
+                    licenses
+                Just licenseName -> do
+                    -- get and show a license`s text
+                    mLicenseText <- getLicense licenseName
+                    case mLicenseText :: Maybe License of
```

Comment:
This function is also shared with the `Project` module, you can put it into the separate function as well 🙂